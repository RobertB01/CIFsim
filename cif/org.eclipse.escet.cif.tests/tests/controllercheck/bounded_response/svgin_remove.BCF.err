WARNING: The specification contains CIF/SVG input declarations. These will be ignored.
WARNING: The alphabet of the specification contains no uncontrollable events.
