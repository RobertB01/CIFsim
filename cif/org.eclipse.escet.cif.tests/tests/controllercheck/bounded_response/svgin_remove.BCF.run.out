Loading CIF specification "controllercheck/bounded_response/svgin_remove.cif"...
Preparing for BDD-based checks...

Restricting system behavior using state/event exclusion plant invariants.

Preparing for MDD-based checks...
Analyzing automaton "p"...
    Initializing the automaton data for event "c_e"...
    Processing edges from the location of automaton "p"...
    Updating global guards and updates for event "c_e"...

Checking for bounded response...
Computing reachable states...
Reachable states: true [initial states predicate]
Forward reachability: iteration 1.

Computing bound for uncontrollable events...
Bounded response check round 1 (states before round: true).

Computing bound for controllable events...
Bounded response check round 1 (states before round: true).

Bounded response check completed.

Checking for finite response...

Iteration 1.
    The following events have been encountered in a controllable-event loop of automaton p:
        (c_e), which is not controllable unconnectable.

Checking for confluence...

CONCLUSION:
    [ERROR] The specification does NOT have bounded response:
    
        - No transitions are possible for uncontrollable events.
        - An infinite sequence of transitions is possible for controllable events.

    [ERROR] The specification may NOT have finite response:
    
        At least one controllable-event loop was found.
        The following events might still occur in a controllable-event loop:
            - c_e

    [OK] The specification has confluence.

The model with the check results has been written to "controllercheck/bounded_response/svgin_remove.BCF.out.cif.real".
